### PR TITLE
Add msgpack dependency

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

**Comment by adamruzicka** [comment]
Incomplete for now. Once we settle this is good to go, I'll copy the package over to focal

**Comment by evgeni** [comment]
[test deb]

File: dependencies/buster/msgpack/control [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Source: msgpack
+Section: ruby
+Priority: optional
+Maintainer: Debian Ruby Extras Maintainers <pkg-ruby-extras-maintainers@lists.alioth.debian.org>
+Uploaders: Adam Ruzicka <aruzicka@redhat.com>
```
Comment (evgeni) [inline dependencies/buster/msgpack/control:5]:
```suggestion
Maintainer: Adam Ruzicka <aruzicka@redhat.com>
```

We are not the Debian Ruby team ;)

File: dependencies/buster/msgpack/control [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+Build-Depends: debhelper (>= 11~),
+               gem2deb,
+               rake,
+               ruby-rspec
+Standards-Version: 4.3.0
+Vcs-Git: https://salsa.debian.org/ruby-team/msgpack.git
+Vcs-Browser: https://salsa.debian.org/ruby-team/msgpack
```
Comment (evgeni) [inline dependencies/buster/msgpack/control:12]:
```suggestion
```

**Comment by evgeni** [comment]
[test deb]

**Comment by evgeni** [comment]
The tests are failing with 

    undefined method `execute' for #<MessagePack::Unpacker:…>`

File: dependencies/buster/msgpack/ruby-tests.rake [hunk 3]
```diff
@@ -0,0 +1,5 @@
+require 'gem2deb/rake/spectask'
+
+Gem2Deb::Rake::RSpecTask.new do |spec|
+  spec.pattern = './spec/**/*_spec.rb'
```
Comment (evgeni) [inline dependencies/buster/msgpack/ruby-tests.rake:4]: this needs to catch `./spec/*_spec.rb` and `./spec/cruby/*_spec.rb` but not `./spec/jruby/*_spec.rb`

**Review by evgeni — CHANGES_REQUESTED** [review]

File: dependencies/buster/msgpack/ruby-tests.rake (hunk 3 above)
Comment (adamruzicka) [reply dependencies/buster/msgpack/ruby-tests.rake]: Done

**Comment by evgeni** [comment]
LGTM

can you copy over to `focal` and `bullseye` please?

File: dependencies/bullseye/msgpack/changelog [hunk 4]
```diff
@@ -0,0 +1,5 @@
+msgpack (1.4.2-1) UNRELEASED; urgency=medium
```
Comment (evgeni) [inline dependencies/bullseye/msgpack/changelog:1]: err, one thing I didn't notice. the package needs to be called `ruby-msgpack` in the changelog

sorry!

File: dependencies/bullseye/msgpack/control [hunk 5]
```diff
@@ -0,0 +1,22 @@
+Source: msgpack
```
Comment (evgeni) [inline dependencies/bullseye/msgpack/control:1]:
```suggestion
Source: ruby-msgpack
```

File: dependencies/bullseye/msgpack/control [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+               gem2deb,
+               rake,
+               ruby-rspec
+Standards-Version: 4.3.0
+Homepage: http://msgpack.org/
+Testsuite: autopkgtest-pkg-ruby
+XS-Ruby-Versions: all
+Rules-Requires-Root: no
+
+Package: msgpack
```
Comment (evgeni) [inline dependencies/bullseye/msgpack/control:15]:
```suggestion
Package: ruby-msgpack
```

**Review by evgeni — CHANGES_REQUESTED** [review]
me---------------

`d/control` also needs the right name, sorry

**Review by evgeni — APPROVED** [review]

**Pull request merged**